.pre-commit-config.yaml

```
@@ -45,7 +45,7 @@ repos:
       exclude: ^tests/.*$
       additional_dependencies:
       - "types-requests"
-      - "pydantic"
+      - "pydantic>=1.9.0,<2"
```

```suggestion
      - "pydantic<2"
```
I think, since we don't want to keep updating the minimum version, keeping it similar to the requirements file, I'd just put less than 2 here instead. This will ensure it will be the latest v1 minor and patch version that is installed, always.

Simultaneously, an issue should be raised to update pydantic support in the code base to v2, removing the version restriction from this line (or maybe setting it to `>=2` instead?)